carbon_black_cloud/plugin.spec.yaml

```diff
@@ -0,0 +1,453 @@
+      description: "Rooted by sensor"
+      required: false
+    rootedBySensorTime:
+      title: "Rooted by Sensor Time"
+      type: string
+      description: "Rooted by sensor time"
+      required: false
+    scanLastActionTime:
+      title: "Scan Last Action Time"
+      type: integer
+      description: "Scan last action time"
+      required: false
+    scanLastCompleteTime:
+      title: "Scan Last Complete Time"
+      type: integer
+      description: "Scan last complete time"
+      required: false
+    scanStatus:
+      title: "Scan Status"
+      type: string
+      description: "Scan status"
+      required: false
+    sensorKitType:
+      title: "Sensor Kit Type"
+      type: string
+      description: "Sensor kit type"
+      required: false
+    sensorOutOfDate:
+      title: "Sensor Out of Date"
+      type: boolean
+      description: "Sensor out of date"
+      required: false
+    sensorPendingUpdate:
+      title: "Sensor Pending Update"
+      type: boolean
+      description: "Sensor pending update"
+      required: false
+    sensorStates:
+      title: "Sensor States"
+      type: "[]string"
+      description: "Sensor states"
+      required: false
+    sensorVersion:
+      title: "Sensor Version"
+      type: string
+      description: "Sensor version"
+      required: false
+    status:
+      title: "Status"
+      type: string
+      description: "Status"
+      required: false
+    targetPriorityType:
+      title: "Target Priority Type"
+      type: string
+      description: "Target priority type"
+      required: false
+    testId:
+      title: "Test ID"
+      type: integer
+      description: "Test ID"
+      required: false
+    uninstallCode:
+      title: "Uninstall Code"
+      type: string
+      description: "Uninstall code"
+      required: false
+    uninstalledTime:
+      title: "Uninstalled Time"
+      type: string
+      description: "Uninstalled time"
+      required: false
+    vdiBaseDevice:
+      title: "VDI Base Device"
+      type: string
+      description: "VDI base device"
+      required: false
+    virtualMachine:
+      title: "Virtual Machine"
+      type: boolean
+      description: "Virtual machine"
+      required: false
+    virtualizationProvider:
+      title: "Virtualization Provider"
+      type: string
+      description: "Virtualization provider"
+      required: false
+    windowsPlatform:
+      title: "Windows Platform"
+      type: string
+      description: "Windows platform"
+      required: false
+
+connection:
+  api_id:
+    title: API ID
+    type: string
+    description: API ID
+    example: ADFF2QLIIZ
+    required: true
+    order: 1
+  api_secret_key:
+    title: API Secret Key
+    type: credential_secret_key
+    description: API secret key
+    example: Z1PXFRDZI321LXQVAB9IJKKZ
+    required: true
+    order: 2
+  url:
+    title: URL
+    type: string
+    description: API URL
+    default: https://api5.conferdeploy.net/integrationServices/v3/
```

Comment:
Can we build this so the user only needs the API base URL as defined in the API URL documentation `https://api5.conferdeploy.net`  https://community.carbonblack.com/t5/Knowledge-Base/PSC-What-URLs-are-used-to-access-the-APIs/ta-p/67346

if they pass in more, we strip it.